integritybackend/actions.py

```diff
@@ -658,235 +673,100 @@ def copy_proofmode(self, zip_path: str, org_config: dict, collection_id: str):
-
-    #     return internal_asset_file
-
-    # def _update(self, asset_fullpath, claim, output_dir, asset_helper):
-    #     # Create temporary files to work with.
-    #     tmp_asset_file = asset_helper.get_tmp_file_fullpath(".jpg")
-    #     tmp_claim_file = asset_helper.get_tmp_file_fullpath(".json")
-
-    #     # Parse file name to get internal name for parent file.
-    #     file_name, file_extension = os.path.splitext(os.path.basename(asset_fullpath))
-    #     parent_file = os.path.join(
-    #         asset_helper.get_assets_internal(), # removed
-    #         file_name.partition("_")[0].partition("-")[0] + file_extension,
-    #     )
-
-    #     # Inject update claim and read back from file.
-    #     time.sleep(1)
-    #     _logger.info("File size: %s", os.path.getsize(asset_fullpath))
-    #     shutil.copy2(asset_fullpath, tmp_asset_file)
-    #     _logger.info("Searching for parent file: %s", parent_file)
-    #     if not os.path.isfile(parent_file):
-    #         raise Exception(f"Expected {parent_file} to be a file, but it isn't")
-
-    #     _logger.info("_update() action found parent file for asset: %s", parent_file)
-    #     _c2patool.run_claim_inject(claim, tmp_asset_file, parent_file)
-    #     _c2patool.run_claim_dump(tmp_asset_file, tmp_claim_file)
-    #     # Copy the C2PA-injected asset to both the internal and shared asset directories.
-    #     internal_asset_file = asset_helper.get_internal_file_fullpath(tmp_asset_file)
-    #     shutil.move(tmp_asset_file, internal_asset_file)
-    #     if output_dir is not None:
-    #         shutil.copy2(internal_asset_file, output_dir)
-    #     _logger.info("New asset file added: %s", internal_asset_file)
-    #     internal_claim_file = asset_helper.get_internal_claim_fullpath(
-    #         internal_asset_file
-    #     )
-    #     shutil.move(tmp_claim_file, internal_claim_file)
-    #     _logger.info(
-    #         "New claim file added to the internal claims directory: %s",
-    #         internal_claim_file,
-    #     )
-    #     return internal_asset_file
-
-    # def _load_custom_assertions(self):
-    #     """Loads custom assertions from file.
-
-    #     Return:
-    #         a dictionary with custom assertions mapped to asset name
-    #     """
-    #     custom_assertions_dictionary = {}
-    #     custom_assertions_path = config.CUSTOM_ASSERTIONS_DICTIONARY
-    #     try:
-    #         with open(custom_assertions_path, "r") as f:
-    #             custom_assertions_dictionary = json.load(f)
-    #             _logger.info(
-    #                 "Successfully loaded custom assertions dictionary: %s",
-    #                 custom_assertions_path,
-    #             )
-    #     except Exception as err:
-    #         _logger.info(
-    #             "No custom assertions dictionary found: %s",
-    #             custom_assertions_path,
-    #         )
-    #     return custom_assertions_dictionary
+    def c2pa_starling_capture(
```

Comment:
Can you add to the main README the set of permitted actions and what they do? I understand there are four right?
```
   "archive": ArchiveHandler,
    "c2pa-proofmode": C2paProofmodeHandler,
    "copy-proofmode": CopyProofmodeHandler,
    "c2pa-starling-capture": C2paStarlingCaptureHandler,
```